libraries/stdlib/jdk8/src/kotlin/random/jdk8/PlatformThreadLocalRandom.kt

```diff
@@ -8,7 +8,7 @@ package kotlin.random.jdk8
 import java.util.concurrent.ThreadLocalRandom
 
 @Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER", "CANNOT_OVERRIDE_INVISIBLE_MEMBER")
-internal class PlatformThreadLocalRandom : kotlin.random.AbstractPlatformRandom() {
+internal object PlatformThreadLocalRandom : kotlin.random.AbstractPlatformRandom() {
```

Comment:
I don't see much sense in transforming it to `object`. It is instantiated once and then is kept in a private property of `Random` companion object. So no need to have another instance where its reference is stored.
Same for `FallbackThreadLocalRandom`